wallet/src/de/schildbach/wallet/WalletApplication.java

```diff
@@ -834,17 +834,6 @@ public int maxConnectedPeers() {
         return isLowRamDevice() ? 4 : 6;
     }
 
-    /**
-     * Low memory devices (currently 1GB or less) and 32 bit devices will require
-     * fewer scrypt hashes on the PIN+salt (handled by dashj)
-     *
-     * @return The number of scrypt interations
-     */
-    public int scryptIterationsTarget() {
```

Comment:
Moved to the `SecurityFunctions` to remove dependency of the callers on the WalletApplication.